corehq/apps/toggle_ui/views.py

```diff
@@ -149,6 +164,43 @@ def post(self, request, *args, **kwargs):
             clear_toggle_cache(toggle.slug, item)
 
         data = {
-            'item_list': item_list
+            'items': item_list
         }
+        if self.usage_info:
+            data['last_used'] = _get_usage_info(toggle)
         return HttpResponse(json.dumps(data), content_type="application/json")
+
+
+def _get_usage_info(toggle):
+    DATE_FORMAT = "%Y-%m-%d"
+    last_used = {}
+    for enabled in toggle.enabled_users:
+        name = _enabled_item_name(enabled)
+        try:
+            if _get_namespace(enabled) == NAMESPACE_DOMAIN:
+                last_used[name] = Domain.get_by_name(name).last_modified.strftime(DATE_FORMAT)
+            else:
+                last_used[name] = CouchUser.get_by_username(name).last_login.strftime(DATE_FORMAT)
+        except (ResourceNotFound, AttributeError):
+            last_used[name] = "Not Found"
+    try:
+        latest_used = sorted(last_used, key=last_used.get, reverse=True)[0]
+    except IndexError:
+        latest_used = None
```

Comment:
nitpick: do you think it would be clearer to use a conditional here rather than `try`/`except`?
```py
if last_used:
    latest_used = sorted(last_used, key=last_used.get, reverse=True)[0]
else:
    latest_used = None
```
Also, I got confused when reading this code because of the similarity in the names `last_used` and `latest_used`. Consider renaming `latest_used` to `most_recent` (something completely different).